### ISSUE TITLE
pg_dump can't connect to docker postgres when not specifying explicit host

### ISSUE THREAD

**Issue by andycmaj** [sdelements/flan#14]
repro:

1. create postgres docker container
2. try to connect using flan

```
❯ flan save minimal-sanjuans
output file name: minimal-sanjuans
    Error: Command failed with exit code 1: pg_dump --dbname=botany --jobs=6 --compress=9 --format=directory --clean --no-owner -U postgres 
    --file=/Users/ac/Workspace/botany/botany/.flan/local/minimal-sanjuans
    pg_dump: error: connection to database "botany" failed: could not connect to server: No such file or directory
        Is the server running locally and accepting
        connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
```

however, if command explicitly specifies host, connection works:

```
pg_dump --dbname=botany --jobs=6 --compress=9 --format=directory --clean --no-owner -U postgres --file=/Users/ac/Workspace/botany/botany/.flan/local/minimal-sanjuans --host=localhost
```

**Comment by andycmaj** [comment]
i have a PR ready if you'd like me to make this change :)

**Comment by hhaidar** [comment]
ah darn, we missed adding `--host`.

If you open a PR that would be awesome, thank you!

**Comment by andycmaj** [comment]
sure! agree with the PGPASSWORD env requirement. especially if you're showing commands executed in the logs :)

but yeah try to get a PR up!

**Comment by hhaidar** [comment]
Published  `0.1.0-alpha.4` to npm.

Thanks you @andycmaj
